### DISCUSSION TITLE
How to write a file through Apache Airflow?

### DISCUSSION THREAD

**Discussion by tanujdhiman** [apache/airflow#17461] [Q&A]
Hello everybody !!

I have made a Dag which has a task of downloading file and save that file. But when I am going to see the directory where I've saved then I found nothing here. So Can anybody tell me how to write file ?

Like Steps for this.
Thanks !!!

**Comment by potiuk** [comment]
You cannot pass local files between tasks. Each task can run on a different machine. You can use XCOM to pass small amount of (usually meta-) data between tasks. If you have custom backends (like S3/GCS backed) you can pass bigger amounts of data as well.

See https://airflow.apache.org/docs/apache-airflow/stable/concepts/xcoms.html

**Reply by tanujdhiman** [reply]
Can you give any example ?
So that I'll be more accurate what I am understanding.

**Reply by potiuk** [reply]
You can watch this talk from Airflow Summit. It has plenty of examples and explains how to do it: https://airflowsummit.org/sessions/2021/customizing-xcom-to-enhance-data-sharing-between-tasks/

**Reply by tanujdhiman** [reply]
This is resolved but getting another error of installing `Feedparser` module so that I can use it.

I tried with `pip install feedparser` but failed .
OS - WSL Ubuntu 18.04

Thanks
